recipes/tclap/all/conanfile.py

```diff
@@ -1,30 +1,48 @@
-from conans import ConanFile, tools
-
 import os
 
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
+
+
 class TclapConan(ConanFile):
     name = "tclap"
+    description = "Templatized Command Line Argument Parser"
     license = "MIT"
-    homepage = "http://github.com/xguerin/tclap"
     url = "https://github.com/conan-io/conan-center-index"
-    description = "Templatized Command Line Argument Parser"
-    topics = ("c++", "commandline parser")
+    homepage = "http://github.com/xguerin/tclap"
+    topics = ("c++", "commandline parser", "header-only")
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 11
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
```

Comment:
Is this really necessary? Checking the sources I could not find any obvious c++11 requirement